docs/lab/gce/ooto-driver.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
+kind: Module
+metadata:
+  name: kmm-ci-a
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: kmm-ci-a 
+
+      kernelMappings:
+        - literal: 5.10.109 
+          containerImage: gcr.io/myproject/kmmo:gke-kmm-1.0 
+          build:
+            buildArgs:
+              - name: CI_BUILD_ARG
+                value: some-build-arg
+            dockerfile: |
+              ARG KERNEL_VERSION=''
+              FROM debian:stable-slim as builder
+              RUN apt-get update && apt-get install -y bc \
+                  bison \
+                  flex \
+                  libelf-dev \
+                  gnupg \
+                  wget \
+                  git \
+                  make \
+                  bc \
+                  gcc && \
+                  kv=${1:-v"$(uname -r | sed -E 's/\+*$//')"} && \
+                  wget "https://chromium.googlesource.com/chromiumos/third_party/kernel/+archive/$kv.tar.gz" && \
+                  mkdir -p /src/kernel && \
+                  tar xzf "$kv.tar.gz" -C /src/kernel
```

Comment:
Should this be a separate `RUN`?

Reply:
I also think the 4 last lines should be in a separate RUN for caching. (it is more likely to change the the packages installation)